src/Tests/Peachpie.ScriptTests/Peachpie.ScriptTests.csproj

```diff
@@ -26,6 +26,13 @@
     <ProjectReference Include="..\..\Peachpie.Runtime\Peachpie.Runtime.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- Make the PHP scripts appear in Visual Studio's Solution Explorer -->
+    <None Include="..\..\..\tests\**">
+      <Link>Scripts\%(RecursiveDir)%(Filename)%(Extension)</Link>
```

Comment:
can we change it from `Scripts\` to `tests\` so it is consistent with directory structure